tests/OptionParsingTest.hh

```diff
@@ -118,4 +118,9 @@ final class OptionParsingTest extends TestCase {
     list($cli, $_, $_) = self::cli('--string=--');
     expect($cli->stringValue)->toBeSame('--');
   }
+
+  public function testMultipleShortOptionsWithAssignment(): void {
+    $this->expectException(InvalidArgumentException::class);
+    self::cli('-vs=something');
```

Comment:
This mostly looks good - can you add a non-exception test case please?

Reply:
@fredemmott Done

Reply:
https://github.com/facebook/hhvm/commit/21e935844cc490d740009c87ce7707a971a1e8e0#diff-84dbfe1742ea02268109f69440abdd8a fixed them